arrows/pdal/pointcloud_io.cxx

```diff
@@ -8,39 +8,25 @@
  */
 
 #include "pointcloud_io.h"
-#include <arrows/pdal/pdal_config.h>
 #include <vital/logger/logger.h>
 #include <vital/exceptions/base.h>
 #include <vital/exceptions/io.h>
 
-#ifdef KWIVER_ENABLE_PDAL
 #include <pdal/PointView.hpp>
 #include <pdal/PointTable.hpp>
 #include <pdal/Dimension.hpp>
 #include <pdal/Options.hpp>
 #include <pdal/StageFactory.hpp>
 
 #include <io/BufferReader.hpp>
-#endif
 
 namespace kwiver {
 namespace arrows {
 namespace pdal {
 
-/// Write landmarks to a file with PDAL provided a geo origin file
-void
-pointcloud_io::save_(vital::path_t const& filename,
-             vital::path_t const& input_geo_origin_file,
-             vital::landmark_map_sptr const& landmarks)
-{
-  auto lgcs = vital::local_geo_cs();
-  read_local_geo_cs_from_file(lgcs, input_geo_origin_file);
-  pointcloud_io::save_(filename, lgcs, landmarks);
-}
-
 /// Write landmarks to a file with PDAL
 void
-pointcloud_io::save_(vital::path_t const& filename,
+save_point_cloud_las(vital::path_t const& filename,
              vital::local_geo_cs const& lgcs,
```

Comment:
Minor formatting nit pick:  If if doesn't make the line lengths too long, I would indent the second and following lines so that the variable types line up with the first line.   The current indention looks arbitrary.  So do this

```c++
save_point_cloud_las(vital::path_t const& filename,
                     vital::local_geo_cs const& lgcs,
                     vital::landmark_map_sptr const& landmarks)
```
or this
```c++
save_point_cloud_las(
  vital::path_t const& filename,
  vital::local_geo_cs const& lgcs,
  vital::landmark_map_sptr const& landmarks)
```

They same formatting rule applies everywhere else you are declaring or defining functions.